checkQC/handlers/reads_per_sample_handler.py

```diff
@@ -32,12 +32,19 @@ def check_qc(self):
         for lane_dict in self.conversion_results:
             lane_nbr = int(lane_dict["LaneNumber"])
             lane_demux = lane_dict["DemuxResults"]
-            nbr_of_samples = len(lane_demux)
+            total_reads = {}
 
             for sample_id_info in lane_demux:
+                sample_name = sample_id_info["SampleName"]
+                if sample_name in total_reads:
+                    total_reads[sample_name] += sample_id_info["NumberReads"] / pow(10, 6)
+                else:
+                    total_reads[sample_name] = sample_id_info["NumberReads"] / pow(10, 6)
+
+            nbr_of_samples = len(total_reads.keys())
+            for sample in total_reads:
```

Comment:
I'd suggest using `for sample, sample_total_reads in total_reads.items(): ...` here to get the key and the value at the same time.

Reply:
Ah ofc, thanks!